src/serlio/serlioPlugin.h

```diff
@@ -20,6 +20,8 @@
 #pragma once
 
 #include <cstdint>
+#include <vector>
+#include <string>
```

Comment:
includes not being in alphabetical order triggers my OCD 😆

Reply:
i'll do some clang-format runs :)